roles/network_plugin/weave/templates/weave-net.yml.j2

```diff
@@ -27,28 +27,28 @@ items:
           - list
           - watch
       - apiGroups:
-          - networking.k8s.io
+          - extensions
```

Comment:
I compared this change with the original manifest of weave-net by downloading it from https://cloud.weave.works/k8s/net?k8s-version=1.23

This apiGroups is `networking.k8s.io`, not `extensions` on the original manifest.
Why here is changed?
I guess this adds apiGroups 'extentions` for networkpolicies API, but I am not sure the reason.

Reply:
I based my changes off of https://github.com/weaveworks/weave/blob/v2.8.1/prog/weave-kube/weave-daemonset-k8s-1.11.yaml I didn't even know the file you linked to was an option.

A quick look shows some of the differences are ordering (which you have commented on below) but some are actual changes like this.

Later on the `networking.k8s.io` is included so this is partly a change and partly an ordering change.

Would you prefer to go off the file you linked to? There are a bunch of additional timestamped annotations in your version that appear to be set based on the time the file is downloaded. Not sure if it makes sense to include those.

Reply:
Thank you for your explanation, I see that.
I confirmed this change is valid by comparing https://github.com/weaveworks/weave/blob/v2.8.1/prog/weave-kube/weave-daemonset-k8s-1.11.yaml
I am fine to apply the above as the original for updating the manifest.

/lgtm